bar-trender-front-end/src/views/ErrorPage.js

```diff
@@ -0,0 +1,73 @@
+import React from "react";
+import DeviceIdentifier from 'react-device-identifier';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/BarTrender/BarTrenderFront/pullRequest?prid=7131439)
```suggestion
import DeviceIdentifier from "react-device-identifier";
```